File: hw/CEDENOR/U3/HW29Composite&TemplateMethod/Java/CompositePattern/src/main/java/ec/edu/espe/compositepattern/controller/Teller.java
package ec.edu.espe.compositepattern.controller;

/**
 *
 * @author Nahomi Cedeño, codecreators, DCCO - ESPE
 */
public class Teller extends Employee {
    public Teller(String aName) {
        this();
        name = aName;
    }

    public Teller() {
        super();
        title = "Teller";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}

File: hw/CEDENOR/U3/HW29Composite&TemplateMethod/Python/Composite/controller/Teller.py
import sys
import os
# Nahomi Cedeño, codecreators, DCCO - ESPE

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Employee import Employee


class Teller(Employee):
    def __init__(self, name):
        super().__init__(name, "Teller")

    def add(self, employee):
        pass  # No operation for leaf nodes

    def remove(self, employee):
        pass  # No operation for leaf nodes